src/core/tests/conditional_compilation/ngraph_cc_off.cpp

```diff
@@ -21,17 +24,28 @@ using namespace std;
 TEST(conditional_compilation, op_scope_with_disabled_cc) {
     int n = 0;
 
-    // Simple scope is enabled
+    // Simple Scope0 is enabled
     OV_OP_SCOPE(Scope0);
     n = 42;
     EXPECT_EQ(n, 42);
 
-    // Simple scope is disabled
+    // Simple Scope1 is enabled regardless of macros
     OV_OP_SCOPE(Scope1);
     n = 43;
     EXPECT_EQ(n, 43);
 }
 
+TEST(conditional_compilation, all_ops_enabled_in_opset) {
+    ov::OpSet opset("test_opset2");
```

Comment:
I propose to add: `#define ov_opset_test_opset3_Abs 1` in order to check that macro doesn't change behavior